### DISCUSSION TITLE
Could you register "<cmd>CodeCompanionActions<Cr>" as an actual telescope extension please?

### DISCUSSION THREAD

**Discussion by Ajaymamtora** [olimorris/codecompanion.nvim#248] [Ideas]
The default layout I use for telescope is a full screen picker, but for things that I know wont have many entries, I pass in a different layout option, but I cant do this if the extension hasnt been registered with telescope:

![image](https://github.com/user-attachments/assets/683e9f38-e4ff-4b44-9070-130f75b6b9f6)

For example your persisted plugin registers the extension with telescope so I can pass in options:

![image](https://github.com/user-attachments/assets/6fcbe669-598b-4b2f-8c01-9f94eda595ae)

**Comment by olimorris** [comment]
This won't be something I'll add but open to a PR so please do investigate.

**Reply by Ajaymamtora** [reply]
https://github.com/olimorris/codecompanion.nvim/pull/250

I've got this so far which seems to work nicely, let me know what you think
